integrations-examples/pipecat-bot/main.py

```
@@ -0,0 +1,103 @@
+import os
+
+from pipecat.frames.frames import Frame, TranscriptionFrame, InterimTranscriptionFrame
+from pipecat.pipeline.pipeline import Pipeline
+from pipecat.pipeline.runner import PipelineRunner
+from pipecat.pipeline.task import PipelineParams, PipelineTask
+from pipecat.processors.frame_processor import FrameDirection, FrameProcessor
+from pipecat.processors.frameworks.rtvi import (
+    RTVIConfig,
+    RTVIObserver,
+    RTVIProcessor,
+    RTVIObserverParams,
```

Ruff (0.12.2)

`pipecat.processors.frameworks.rtvi.RTVIObserverParams` imported but unused

Remove unused import: `pipecat.processors.frameworks.rtvi.RTVIObserverParams`

(F401)